lib/xnvmec.c

```diff
@@ -939,6 +939,33 @@ static struct xnvmec_opt_attr xnvmec_opts[] = {
 		.name = "\nWith <args> for backend:\n",
 		.descr = "",
 	},
+
```

Comment:
The empty lines here should be removed to follow the convention of the previous blocks

Reply:
I'm not sure I follow. We make new-lines in this list at line 887 and 880.